### DISCUSSION TITLE
Examples

### DISCUSSION THREAD

**Discussion by alyoung1621** [atlassian/pragmatic-drag-and-drop#83] [Q&A] [answered]
On the Atlassian examples page, the code is provided via CodeSandbox. Per [CodeSandbox](https://sandpack.codesandbox.io/docs/resources/faq#what-license-do-sandpack-and-nodebox-use), Nodebox cannot be used for commercial use. Is it possible to make the source code available for each example in separate repos? The documentation package [README](https://github.com/atlassian/pragmatic-drag-and-drop/tree/main/packages/documentation), containing the examples folder, indicates it is not intended for direct use by consumers.

**Comment by alexreardon** [comment, marked as answer]
Interesting! You can find all the examples in the documentation package. I should cleanup the warning about the documentation package. What it is meaning to say is that the documentation package is not designed to be installed and run like our other packages. It's mostly just a bucket for examples.

My understanding (which might be wrong) is that a review was performed a while ago about the usage of codesandbox on our site. Keep in mind that we are freely licensing all of the Atlassian Design System and Pragmatic drag and drop with the permissive Apache 2.0 license
